packages/scripts/README.md

```diff
@@ -271,17 +271,9 @@ The `build` and `start` commands use [webpack](https://webpack.js.org/) behind t
 
 * [Entry](https://webpack.js.org/configuration/entry-context/#entry): `src/index.js`
 * [Output](https://webpack.js.org/configuration/output): `build/index.js`
-* [Externals](https://webpack.js.org/configuration/externals). These are libraries that are to be found in the global scope:
-
-Package | Input syntax | Output
```

Comment:
Can we keep this list of externals but move it to the webpack plugin docs?

Reply:
I added the following:

By default, the following module requests are handled:

> | Request | Global | Script handle |
> | --- | --- | --- |
> | `@babel/runtime/regenerator` | `regeneratorRuntime` | `wp-polyfill` |
> | `@wordpress/*` | `wp.*` | `wp-*` |
> | `jquery` | `jQuery` | `jquery` |
> | `jquery` | `jQuery` | `jquery` |
> | `lodash-es` | `lodash` | `lodash` |
> | `lodash` | `lodash` | `lodash` |
> | `moment` | `moment` | `moment` |
> | `react-dom` | `ReactDOM` | `react-dom` |
> | `react` | `React` | `react` |